README.md

```diff
@@ -45,10 +106,11 @@ For bug reports, please [open an issue on GitHub](https://github.com/rollbar/rol
 
 To set up a development environment, you'll need Node.js and npm.
 
-2. `npm install -D`
-3. `make`
+1. Clone the repository: `git clone https://github.com/rollbar/rollbar.js.git`
+2. Install dependencies: `npm install`
+3. Build: `npm run build`
 
-To run the tests, run `make test`.
+To run the tests: `npm test`
```

Comment:
Don't know why do we even have a `Makefile`, do js devs don't know how to use `npm`? 😂

I was tempted to remove the Makefile, but I'm not sure if its being used by the CI or something else.